xml/Microsoft.Extensions.Http.Logging/IHttpClientLogger.xml

```diff
@@ -1,139 +1,145 @@
-      <Parameters>
-        <Parameter Name="context" Type="System.Object">
-          <Attributes>
-            <Attribute>
-              <AttributeName Language="C#">[System.Runtime.CompilerServices.Nullable(2)]</AttributeName>
-              <AttributeName Language="F#">[&lt;System.Runtime.CompilerServices.Nullable(2)&gt;]</AttributeName>
-            </Attribute>
-          </Attributes>
-        </Parameter>
-        <Parameter Name="request" Type="System.Net.Http.HttpRequestMessage" />
-        <Parameter Name="response" Type="System.Net.Http.HttpResponseMessage" />
-        <Parameter Name="elapsed" Type="System.TimeSpan" />
-      </Parameters>
-      <Docs>
-        <param name="context">To be added.</param>
-        <param name="request">To be added.</param>
-        <param name="response">To be added.</param>
-        <param name="elapsed">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
-      </Docs>
-    </Member>
-  </Members>
-</Type>
+<Type Name="IHttpClientLogger" FullName="Microsoft.Extensions.Http.Logging.IHttpClientLogger">
+  <TypeSignature Language="C#" Value="public interface IHttpClientLogger" />
+  <TypeSignature Language="ILAsm" Value=".class public interface auto ansi abstract IHttpClientLogger" />
+  <TypeSignature Language="DocId" Value="T:Microsoft.Extensions.Http.Logging.IHttpClientLogger" />
+  <TypeSignature Language="VB.NET" Value="Public Interface IHttpClientLogger" />
+  <TypeSignature Language="F#" Value="type IHttpClientLogger = interface" />
+  <TypeSignature Language="C++ CLI" Value="public interface class IHttpClientLogger" />
+  <AssemblyInfo>
+    <AssemblyName>Microsoft.Extensions.Http</AssemblyName>
+    <AssemblyVersion>8.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <Interfaces />
+  <Attributes>
+    <Attribute>
+      <AttributeName Language="C#">[System.Runtime.CompilerServices.NullableContext(1)]</AttributeName>
+      <AttributeName Language="F#">[&lt;System.Runtime.CompilerServices.NullableContext(1)&gt;]</AttributeName>
+    </Attribute>
+  </Attributes>
+  <Docs>
+    <summary>An abstraction for custom HTTP request logging for a named <see cref="T:System.Net.Http.HttpClient" /> instances returned by <see cref="T:System.Net.Http.IHttpClientFactory" />.</summary>
+    <remarks>
+      <para> It is up to the user implementing the interface to decide where (to <see cref="T:Microsoft.Extensions.Logging.ILogger" />, or anything else) and what exactly to log.
+            However, the implementation should be mindful about potential adverse side effects of accessing some of the <see cref="T:System.Net.Http.HttpRequestMessage" /> or <see cref="T:System.Net.Http.HttpResponseMessage" /> properties, such as reading from a content stream; if possible, such behavior should be avoided.
+            </para>
+      <para> Logging implementation also should not throw any exceptions, as an unhandled exception in logging would fail the request.

```

Comment:
```suggestion
      <para>The logging implementation shouldn't throw any exceptions, as an unhandled exception in logging would fail the request.
```